build-llvm11.md

```diff
@@ -7,18 +7,21 @@ slug: getting-started
 
 {% include version_warning.md %}
 
-The current procedure for building KLEE with LLVM 11 (recommended) is outlined below.
+The current procedure for building KLEE manually with LLVM 11 (recommended) on Debian/Ubuntu-based distributions or macOS is outlined below.
+However, in case you trust our installation scripts for continuous integration testing, you can re-use them on your Ubuntu-based host system.
+You can find instructions at the end of this page.
 
+## Manual Installation
 
 **NOTE:** KLEE is currently tested on Linux x86-64 (particularly Ubuntu), FreeBSD and macOS.
 There is no support for uClibc and the POSIX environment under macOS.
 KLEE does not work under x86-32.
 
-1. **Install dependencies:** KLEE requires all the dependencies of LLVM (see [here](http://llvm.org/docs/GettingStarted.html#requirements)), and some more. In particular, you should install the programs and libraries listed below.
+1. **Install dependencies:** KLEE requires all the dependencies of LLVM (see [here](http://llvm.org/docs/GettingStarted.html#requirements)), and some more. In particular, you should install the programs and libraries listed below. `doxygen` is optional and only needed to generate the source code documentation.
```

Comment:
Can you also add `graphviz` here (optional as `doxygen`, although I don't think we need to say which ones are optional, especially since the list is likely incomplete, depending on what features you use)

Reply:
> the list is likely incomplete

Should be complete imho, we mention further down that the tcmalloc packages are not required for glibc malloc.

Reply:
Yes, but by the time you get there, it's too late.  I personally won't make this distinction, but if you insist...

Reply:
I'd consider tcmalloc the default setting opposed to doxygen.

Reply:
Yes, but both are optional ;)   
My point is that it's not that important to make this distinction, and I'd rather keep the instructions short.  Regardless, if you'd really like to add that extra sentence, it's fine with me. 

You forgot to add graphviz to the macOS line

Reply:
> you'd really like to add that extra sentence

They pull in quite a lot of dependencies (250MiB in Docker), hence I'd keep it.

> You forgot to add graphviz to the macOS line

I just can't test it and don't care about the crappy OS, but ok.

Reply:
Well, even if you don't use -- or like -- all the OSes supported by KLEE, you still need to make sure any changes cater for all supported platforms.  `brew` is supported on Linux too, btw.